popos.py

```diff
@@ -1,11 +1,11 @@
 class Book:
 	def __init__(self, object):
 		self.title = object['volumeInfo'].setdefault('title', '')
-		self.author = object['volumeInfo'].setdefault('authors', [])
+		self.authors = '::'.join(object['volumeInfo'].setdefault('authors', []))
```

Comment:
Great. Will set up a conditional to check for this syntax wen displaying books on the single book component page

Reply:
Updated PR now returns authors and genres as an array. Won't need to do any string parsing!